v2/pkg/passive/sources.go

```diff
@@ -83,6 +84,7 @@ var AllSources = [...]subscraping.Source{
 	&whoisxmlapi.Source{},
 	&zoomeye.Source{},
 	&zoomeyeapi.Source{},
+	&leakix.Source{},
```

Comment:
You should probably move this under line 72. It's causing `subfinder -ls` to display leakix in the wrong order of the sources list (not sorted sort alphabetically).

Reply:
Good catch, PR opened